dutchx_scripts/daoCreateDutchX.ts

```diff
@@ -0,0 +1,227 @@
+import {
+  Address,
+  Auction4ReputationFactory,
+  ConfigService,
+  DAO,
+  ExternalLocking4ReputationFactory,
+  InitializeArcJs,
+  LockingEth4ReputationFactory,
+  LockingToken4ReputationFactory,
+  Utils,
+  Web3
+} from "@daostack/arc.js";
+
+import { run as contractNew } from "../scripts/contractNew";
+import { run as daoCreate } from "../scripts/daoCreate";
+import { run as tokenMint } from "../scripts/tokenMint";
+
+export const run = async (web3: Web3, networkName: string): Promise<void> => {
+
+// tslint:disable: max-line-length
+
+  await InitializeArcJs({
+    filter: {},
+  });
+
+  ConfigService.set("estimateGas", true);
+
+  const daoSchema = require(`../../dutchx_scripts/specsNewDaos/dutchX.${networkName.toLowerCase()}.json`);
+
+  const lockingEth4Reputation =
+    await LockingEth4ReputationFactory.at((await contractNew(web3, networkName, { name: "LockingEth4Reputation" }, "max", "50000000000") as Lock4ReputationContract).address);
```

Comment:
Want to do the config file idea in a separate PR, otherwise this PR is going to be held up, unless we make the config file idea higher priority than working the UI.  Your call.

Reply:
we need to have a clean deployment scripts.
lets do it.

Reply:
https://github.com/daostack/dx-deployment-script/pull/1/commits/a376b5a0bc59d8546654d35d67903874741d8db3